langfuse-node/src/langfuse-node.ts

```diff
@@ -26,8 +27,10 @@ export default class Langfuse extends LangfuseCore {
   private options: LangfuseOptions;
 
   constructor(params: { publicKey: string; secretKey: string } & LangfuseOptions) {
-    const { publicKey, secretKey, ...options } = params;
     super(params);
+
+    const { secretKey, ...options } = utils.configLangfuseSDK(params);
```

Comment:
same error here:

```
Property 'configLangfuseSDK' does not exist on type 'typeof utils_d'.ts(2339)

```

Reply:
We have so far been unable to reproduce this error, can you please share some more context on this?